### DISCUSSION TITLE
consistency with other sites?

### DISCUSSION THREAD

**Discussion by tpd-opitz** [electricitymaps/electricitymaps-contrib#2865] [General]
I wonder, why your data for Germany  differ from the data presented a the [agorameter](https://www.agora-energiewende.de/service/agorameter/chart/power_price_emission/23.12.2020/23.12.2020/).

Eg.: for today (2020/12/23) 13:00 MET your value of CO2 emission for German Electric power production (not consumption) is 396g/kWh while the agorameter shows  462g/kWh which is about 20% higher.

Where does that difference come from?

**Comment by veqtrus** [comment]
ElectricityMap uses IPCC median emission intensities in most cases. Germany (and Poland and other countries) burn more brown coal which has higher emissions. #2942
